### DISCUSSION TITLE
Choosing alternate suggestions for copilot.vim

### DISCUSSION THREAD

**Discussion by leahein** [community/community#7053] [Copilot Conversations] [answered]
How do you cycle through alternate suggestions in neovim?

The [VSCode shortcut of `alt + [`](https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#3-choosing-alternate-suggestions) doesn't seem to work, and I can't seem to find anything in the docs.

Thanks so much!

**Comment by tpope** [comment]
Not supported yet but it's on the roadmap!

**Comment by Lucklyric** [comment]
@leahein Feel free to use this fork as a temporary solution. https://github.com/Lucklyric/copilot.vim,

**Comment by tpope** [comment, marked as answer]
Suggestion cycling is supported in the latest release.

**Comment by vhristev** [comment]
Is it working for someone with NeoVim / MacOSX. 

I check my vim maps but still cannot make it work.

**Reply by tpope** [reply]
See [this comment](https://github.com/orgs/community/discussions/6862#discussioncomment-3012543).
